src/main/java/com/example/zawody/ZawodyApplication.java

```diff
@@ -0,0 +1,85 @@
+package com.example.zawody;
+
+
+import com.example.zawody.model.Competition;
+import com.example.zawody.model.Person;
+import com.example.zawody.model.Result;
+import com.example.zawody.repositories.CompetitionRepository;
+import com.example.zawody.repositories.PersonRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.neo4j.repository.config.EnableNeo4jRepositories;
+
+
+@SpringBootApplication
+@EnableNeo4jRepositories
+public class ZawodyApplication{
+	public static void main(String[] args) {
+		SpringApplication.run(ZawodyApplication.class, args);
+	}
+
+	@Bean
+	public ModelMapper modelMapper() {
+		return new ModelMapper();
+	}
+	@Bean
+	CommandLineRunner demo(PersonRepository personRepository, CompetitionRepository competitionRepository) {
```

Comment:
proper usage of this generator, would be to run it based on a an environment in Development it can be run, but on Production it shouldn't be executed, because no one wants to lose all the data